Assets/Scripts/Combat/PossessionManager.cs

```diff
@@ -139,9 +143,10 @@ public void RevertPossession()
 
     void SetEnemyComponentsEnabled(bool value)
     {
-        enemy.enabled = value;
-        enemyAttack.enabled = value;
-        aIMovement.enabled = value;
+        // enemy.enabled = value;
+        // enemyAttack.enabled = value;
+        // aIMovement.enabled = value;
```

Comment:
I'm guessing BaseEnemyAI replaces the above scripts?

Otherwise we could enable / disable conditionally:

```
if (enemy != null) enemy.enabled = value;
if (enemyAttack != null) enemyAttack.enabled = value;
// etc.
```